.github/workflows/ci_cd_nightly.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           echo CPU info
           lscpu
           docker pull ${{ env.MECHANICAL_IMAGE }}
-          BUILD_DATE=$(docker run --rm --entrypoint head ${{ env.MECHANICAL_IMAGE }} -n 1 /install/ansys_inc/v242/aisol/CommonFiles/builddate.txt)
+          BUILD_DATE=$(docker run --rm --entrypoint head ${{ env.MECHANICAL_IMAGE }} -n 1 /install/ansys_inc/v251/aisol/CommonFiles/builddate.txt)
```

Comment:
can we use a variable instead of hardcoding

Reply:
Yes